engine/link/link.asm

```diff
@@ -83,23 +83,23 @@ endc
 	ld hl, wLinkBattleRNPreamble
 	ld de, wEnemyMon
 	ld bc, SERIAL_RN_PREAMBLE_LENGTH + SERIAL_RNS_LENGTH
-	vc_hook Network358
+	vc_hook ExchangeBytes1  ; VC hook that is called right before a routine to Exchange Bytes occurs.
```

Comment:
None of these "`; VC hook that is called...`" comments are necessary, since `vc_hook` and an appropriate name are self-explanatory.